--- seamonkey-2.53.6/comm/mail/app/profile/all-thunderbird.js.orig	2021-03-06 17:28:54.001239823 +0100
+++ seamonkey-2.53.6/comm/mail/app/profile/all-thunderbird.js	2021-03-06 17:48:10.610717566 +0100
@@ -148,7 +148,7 @@
 // Disable add-ons installed into the shared user and shared system areas by
 // default. This does not include the application directory. See the SCOPE
 // constants in AddonManager.jsm for values to use here
-pref("extensions.autoDisableScopes", 15);
+pref("extensions.autoDisableScopes", 0);
 
 // Enable add-ons installed and owned by the application, like the default theme.
 pref("extensions.startupScanScopes", 4);
--- seamonkey-2.53.4/comm/suite/app/profile/suite-prefs.js.orig	2020-08-10 13:28:35.000000000 +0200
+++ seamonkey-2.53.4/comm/suite/app/profile/suite-prefs.js	2020-10-09 07:09:53.453582063 +0200
@@ -581,7 +581,7 @@
 // Disable add-ons installed into the shared user and shared system areas by
 // default. This does not include the application directory. See the SCOPE
 // constants in AddonManager.jsm for values to use here
-pref("extensions.autoDisableScopes", 15);
+pref("extensions.autoDisableScopes", 3);
 
 // Enable add-ons installed and owned by the application, like the default theme.
 pref("extensions.startupScanScopes", 4);
