--- seamonkey-2.53.6/python/mozbuild/mozbuild/virtualenv.py.orig	2020-10-09 08:14:38.869199627 +0200
+++ seamonkey-2.53.6/python/mozbuild/mozbuild/virtualenv.py	2020-10-09 08:30:19.570770076 +0200
@@ -66,8 +66,7 @@
     @property
     def virtualenv_script_path(self):
         """Path to virtualenv's own populator script."""
-        return os.path.join(self.topsrcdir, 'third_party', 'python',
-            'virtualenv', 'virtualenv.py')
+        return '/usr/bin/virtualenv-2'
 
     @property
     def bin_path(self):
@@ -424,7 +424,7 @@
                 handle_package(package)
 
             sitecustomize = os.path.join(
-                os.path.dirname(os.__file__), 'sitecustomize.py')
+                os.path.dirname(distutils.sysconfig.get_python_lib()), 'sitecustomize.py')
             with open(sitecustomize, 'w') as f:
                 f.write(
                     '# Importing mach_bootstrap has the side effect of\n'
