--- seamonkey-2.53.17.1/python/mozbuild/mozbuild/virtualenv.py.orig	2022-08-24 21:55:13.000000000 +0200
+++ seamonkey-2.53.17.1/python/mozbuild/mozbuild/virtualenv.py	2022-11-25 22:13:52.882020703 +0100
@@ -73,8 +73,7 @@ class VirtualenvManager(object):
     @property
     def virtualenv_script_path(self):
         """Path to virtualenv's own populator script."""
-        return os.path.join(self.topsrcdir, 'third_party', 'python',
-                            'virtualenv', 'virtualenv.py')
+        return '/usr/bin/virtualenv-3'
 
     def version_info(self):
         return eval(subprocess.check_output([
